{"uid":"1898b51ffb9e9a31","name":"Валидация поля limit: -1","fullName":"tests.test_backend.board.board_validation_test#test_create_board_group_limit_validation","historyId":"96f74604c2170a1f91a88a047bacd430","time":{"start":1749798751103,"stop":1749798751359,"duration":256},"status":"passed","statusMessage":"XPASS Бэк принимает отрицательное значение","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"delete_output_dir","time":{"start":1749798742635,"stop":1749798742635,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"owner_client","time":{"start":1749798742635,"stop":1749798743126,"duration":491},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"base_url","time":{"start":1749798742635,"stop":1749798742635,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"pytestconfig","time":{"start":1749798742635,"stop":1749798742635,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"_verify_url","time":{"start":1749798742635,"stop":1749798742635,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_space","time":{"start":1749798743126,"stop":1749798744123,"duration":997},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_project","time":{"start":1749798744123,"stop":1749798745984,"duration":1861},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_board","time":{"start":1749798748751,"stop":1749798749138,"duration":387},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"passed","statusMessage":"XPASS Бэк принимает отрицательное значение","steps":[{"name":"Проверка, что API вернул 200 при limit = -1","time":{"start":1749798751359,"stop":1749798751359,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":0,"hasContent":true},"afterStages":[{"name":"temp_space::0","time":{"start":1749798778176,"stop":1749798778980,"duration":804},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(reason='Бэк принимает отрицательное значение')"},{"name":"tag","value":"backend"},{"name":"parentSuite","value":"tests.test_backend.board"},{"name":"suite","value":"board_validation_test"},{"name":"host","value":"pkrvmxyh4eaekms"},{"name":"thread","value":"2537-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_backend.board.board_validation_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"expected_status","value":"200"},{"name":"limit","value":"-1"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":8,"passed":6,"unknown":0,"total":14},"items":[{"uid":"893a838d197207bf","reportUrl":"https://vaizcom.github.io/autotests/128//#testresult/893a838d197207bf","status":"skipped","statusDetails":"XFAIL Бэк принимает отрицательное значение\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f254b41e350>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749798175890,"stop":1749798175890,"duration":0}},{"uid":"51f9117ecd8f607","reportUrl":"https://vaizcom.github.io/autotests/127//#testresult/51f9117ecd8f607","status":"passed","statusDetails":"XPASS Бэк принимает отрицательное значение","time":{"start":1749797899773,"stop":1749797900015,"duration":242}},{"uid":"13ccc2b95e815744","reportUrl":"https://vaizcom.github.io/autotests/126//#testresult/13ccc2b95e815744","status":"passed","statusDetails":"XPASS Бэк принимает отрицательное значение","time":{"start":1749745009631,"stop":1749745009877,"duration":246}},{"uid":"b90e2c21141b1dea","reportUrl":"https://vaizcom.github.io/autotests/125//#testresult/b90e2c21141b1dea","status":"passed","statusDetails":"XPASS Бэк принимает отрицательное значение","time":{"start":1749744408711,"stop":1749744409016,"duration":305}},{"uid":"afb8f3eaad09cc2b","reportUrl":"https://vaizcom.github.io/autotests/124//#testresult/afb8f3eaad09cc2b","status":"passed","statusDetails":"XPASS Бэк принимает отрицательное значение","time":{"start":1749744322005,"stop":1749744322286,"duration":281}},{"uid":"942f5382ccbd0a1f","reportUrl":"https://vaizcom.github.io/autotests/123//#testresult/942f5382ccbd0a1f","status":"skipped","statusDetails":"XFAIL Бэк принимает отрицательное значение\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6ed7455e50>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749744003512,"stop":1749744003512,"duration":0}},{"uid":"20bc350334fd3a87","reportUrl":"https://vaizcom.github.io/autotests/122//#testresult/20bc350334fd3a87","status":"passed","statusDetails":"XPASS Бэк принимает отрицательное значение","time":{"start":1749743625980,"stop":1749743626233,"duration":253}},{"uid":"2a5eddc637109803","reportUrl":"https://vaizcom.github.io/autotests/119//#testresult/2a5eddc637109803","status":"skipped","statusDetails":"XFAIL Бэк принимает отрицательное значение\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fc7b5913490>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742820651,"stop":1749742820651,"duration":0}},{"uid":"bae82b101e3464a6","reportUrl":"https://vaizcom.github.io/autotests/118//#testresult/bae82b101e3464a6","status":"skipped","statusDetails":"XFAIL Бэк принимает отрицательное значение\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9cb88f1f10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742668049,"stop":1749742668049,"duration":0}},{"uid":"9dc176eb8b56f247","reportUrl":"https://vaizcom.github.io/autotests/117//#testresult/9dc176eb8b56f247","status":"skipped","statusDetails":"XFAIL Бэк принимает отрицательное значение\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f2b9f21f250>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742497848,"stop":1749742497848,"duration":0}},{"uid":"7a25effe5b57e396","reportUrl":"https://vaizcom.github.io/autotests/116//#testresult/7a25effe5b57e396","status":"skipped","statusDetails":"XFAIL Бэк принимает отрицательное значение\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7efcc7f78ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749741234253,"stop":1749741234253,"duration":0}},{"uid":"7c7a1e0f31e4c270","reportUrl":"https://vaizcom.github.io/autotests/115//#testresult/7c7a1e0f31e4c270","status":"skipped","statusDetails":"XFAIL Бэк принимает отрицательное значение\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f28d1ddcd10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740826778,"stop":1749740826778,"duration":0}},{"uid":"902e29acc174ec25","reportUrl":"https://vaizcom.github.io/autotests/114//#testresult/902e29acc174ec25","status":"skipped","statusDetails":"XFAIL Бэк принимает отрицательное значение\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f271b110ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740653517,"stop":1749740653517,"duration":0}}]},"tags":["@pytest.mark.xfail(reason='Бэк принимает отрицательное значение')","backend"]},"source":"1898b51ffb9e9a31.json","parameterValues":["200","-1"]}